docs/src/docs/core/Collapse.mdx

```diff
@@ -0,0 +1,41 @@
+---
+group: 'mantine-core'
+package: '@mantine/core'
+title: Collapse
+order: 1
+slug: /core/collapse/
+category: 'data-display'
+description: 'Collapsible util component'
+props: ['Collapse']
+import: "import { Collapse } from '@mantine/core';"
+source: 'mantine-core/src/components/Collapse/Collapse.tsx'
+docs: 'core/Collapse.mdx'
+styles: ['Collapse']
+---
+
+import { CollapseDemos } from '@demos/core';
+
+## Usage
+
+<Demo data={CollapseDemos.usage} />
+
+## Styles API
```

Comment:
Collapse is a very simple component there is no need for Styles API demo and docs, remove it